### PR TITLE
fix bug 1161812 - put revision files in root of package

### PR DESCRIPTION
**Pull request by rhelmer**
r? @peterbe - this is pretty much the simplest way I could figure out.

### CODE REVIEW

File: scripts/install.sh [hunk 1]
```diff
@@ -63,3 +63,7 @@ fi
 if [ "$BUILD_TYPE" != "tar" ]; then
     BUILD_DIR=${BUILD_DIR%%/data/socorro}
 fi
+
+# install socorro in local virtualenv
+# this must run at the end to capture any generated files above
+$BUILD_DIR/socorro-virtualenv/bin/python setup.py install
```
Comment (peterbe) [inline scripts/install.sh]: Wait what?! Did we not do this before?

File: setup.py [hunk 2]
```diff
@@ -54,5 +54,7 @@ def read(fname):
             glob.glob('socorro/external/postgresql/raw_sql/views/*.sql')),
         ('socorro/external/postgresql/raw_sql/types',
             glob.glob('socorro/external/postgresql/raw_sql/types/*.sql')),
+        ('socorro', ['socorro_revision.txt', 'breakpad_revision.txt',
+               'JENKINS_BUILD_NUMBER'])
```
Comment (peterbe) [inline setup.py]: is `JENKINS_BUILD_NUMBER` a file?

File: socorro/external/postgresql/server_status.py [hunk 3]
```diff
@@ -83,13 +83,9 @@ def get(self, **kwargs):
...
 
         # Find the current breakpad and socorro revisions
-        basedir = os.path.dirname(__file__)
-        socorro_revision = open(
-            os.path.join(basedir, 'socorro_revision.txt')
-        ).read().strip()
-        breakpad_revision = open(
-            os.path.join(basedir, 'breakpad_revision.txt')
-        ).read().strip()
+        from pkg_resources import resource_string
```
Comment (peterbe) [inline socorro/external/postgresql/server_status.py]: move to the top?

File: scripts/install.sh (hunk 1 above)
Comment (rhelmer) [inline scripts/install.sh]: > Wait what?! Did we not do this before?

We do, but we need to run it again to get any generated files in there.

**Comment by peterbe** [comment]
Looks good. I'm guessing you didnt't code this out of thin air but that you actually tested it and once it worked you wrote it down in a PR.

File: setup.py (hunk 2 above)
Comment (rhelmer) [inline setup.py]: Yes, gets created by Jenkins.

**Comment by rhelmer** [comment]
> I'm guessing you didnt't code this out of thin air but that you actually tested it and once it worked you wrote it down in a PR.

Yep I tested it locally, need to make sure it can get through our real build process.

File: setup.py (hunk 2 above)
Comment (peterbe) [inline setup.py]: So Travis complained that `socorro_revision.txt` didn't get created.

**Comment by rhelmer** [comment]
Going to land this now so as to unbust AWS stage's `/status` page

**Pull request merged**